### PR TITLE
add features by MarlonPasso-git

### PR DESCRIPTION
**Pull request by MarlonPassos-git**
# Repository
https://github.com/MarlonPassos-git/dev-container-features

# Compatible / Tested 
- mcr.microsoft.com/devcontainers/javascript-node:1-18-bullseye
- mcr.microsoft.com/devcontainers/javascript-node:1-20-bullseye
- mcr.microsoft.com/devcontainers/base:ubuntu

# Features 
- [vtex cli](https://github.com/MarlonPassos-git/dev-container-features/tree/main/src/vtex-cli) 

# Notes 

My company started to use dev containers but missed some implementation of the cli of the tool, which we use every day. Now I have created the feature implementation of VTEX CLI.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -608,3 +608,8 @@
   contact: https://github.com/joshspicer/more-features/issues
   repository: https://github.com/joshspicer/more-features
   ociReference: ghcr.io/joshspicer/more-features
+- name: Features by MarlonPassos-git
+  maintainer: MarlonPassos-git
+  contact: hhttps://github.com/MarlonPassos-git/dev-container-features/issues
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  contact: https://github.com/MarlonPassos-git/dev-container-features/issues
```

File: _data/collection-index.yml [hunk 2]
```diff
@@ -608,3 +608,8 @@
...
+  repository: https://github.com/MarlonPassos-git/dev-container-features
+  ociReference: ghcr.io/MarlonPassos-git/dev-container-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/marlonpassos-git/dev-container-features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

Comment (MarlonPassos-git) [reply _data/collection-index.yml]: Of course, thanks for the feedback

**Review by samruddhikhandale — APPROVED** [review]
Nice, thanks!

**Pull request merged**